docs/developers/relayed-transactions.md

```diff
@@ -0,0 +1,222 @@
+---
+id: relayed-transactions
+title: Relayed Transactions
+---
+
+## Introduction
+
+Relayed transactions (or meta-transactions) are transactions where a so-called relayer pays the fee for another
+transactions. In other words, addresses can interact with Smart Contracts without having any EGLD for fees, when a 
+relayer is willing to pay for that transaction.
+
+More details and specifications can be found on [Elrond Specs](https://github.com/ElrondNetwork/elrond-specs/blob/main/sc-meta-transactions.md).
+
+## Types of relayed transactions
+
+Currently, there are 2 versions of relayed transactions: v1 and v2. In the end, they both have the same effect,
+but v2 comes with optimisations in terms of gas usage, making it our recommendation.
+
+## Relayed transactions version 1
+
+A relayed transaction version 1 relies on having the inner transaction JSON serialized and given as an argument to the `relayedTx` protocol function. 
+
+It would look like: 
+
+```
+RelayedV1Transaction {
+    Sender: <Relayer address>
+    Receiver: <Address that signed the inner transaction>
+    Value: 0
+    GasLimit: <move_balance_cost> + length(Data) * <gas_per_data_byte> + <inner transaction gas limit>
+    Data: "relayedTx" +
+          "@" + <JSON serialized inner transaction in hexadecimal encoding>
+}
+```
+
+The inner transaction can have a format like this:
+
+```
+RelayedV1InnerTransaction {
+    Sender: <Receiver of the relayed transaction>
+    Receiver: <Smart Contract address>
+    Value: 0
+    GasLimit: <to be determined for each case>
+    Data: "functionName" +
+          "@" + <argument in hexadecimal encoding>
+          ...
+}
+```
+
+However, unlike regular transactions' JSON serialization, the inner transaction that has to be signed has a different 
+structure:
+
+```
+type Transaction struct {
+	Nonce             uint64        
+	Value             *math_big.Int 
+	ReceviverAddress  []byte        
+	RcvUserName       []byte        
+	SenderAddress     []byte        
+	SndUserName       []byte        
+	GasPrice          uint64        
+	GasLimit          uint64        
+	Data              []byte        
+	ChainID           []byte        
+	Version           uint32        
+	Signature         []byte        
+	Options           uint32        
+}
+```
+
+Notice that there are some differences as compared to the regular *frontend* [transaction structure](/developers/signing-transactions/signing-transactions/#general-structure), such:
+- sender and receiver have to be byte arrays instead of bech32 addresses strings
+- value has to be a big integer, instead of a string
+- chain ID has to be a byte array instead of a string
+- signature has to be a byte array instead of the hex version of it
```

Comment:
maybe more specific here, like using the exact field name (with capital letters) and code style

Reply:
true, refactored. removed sender and receiver username as they could create confusion